internal/cmd/sqlserverflex/instance/list/list.go

```diff
@@ -0,0 +1,164 @@
+				"$ stackit sqlserverflex instance list --limit 10"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("get SQLServer Flex instances: %w", err)
+			}
+			if resp.Items == nil || len(*resp.Items) == 0 {
+				projectLabel, err := projectname.GetProjectName(ctx, p, cmd)
+				if err != nil {
+					p.Debug(print.ErrorLevel, "get project name: %v", err)
+					projectLabel = model.ProjectId
+				}
+				p.Info("No instances found for project %q\n", projectLabel)
+				return nil
+			}
+			instances := *resp.Items
+
+			// Truncate output
+			if model.Limit != nil && len(instances) > int(*model.Limit) {
+				instances = instances[:*model.Limit]
+			}
+
+			return outputResult(p, model.OutputFormat, instances)
+		},
+	}
+
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Int64(limitFlag, 0, "Maximum number of entries to list")
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	limit := flags.FlagToInt64Pointer(p, cmd, limitFlag)
+	if limit != nil && *limit < 1 {
+		return nil, &errors.FlagValidationError{
+			Flag:    limitFlag,
+			Details: "must be greater than 0",
+		}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		Limit:           limit,
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *sqlserverflex.APIClient) sqlserverflex.ApiListInstancesRequest {
+	req := apiClient.ListInstances(ctx, model.ProjectId)
+	return req
+}
+
+func outputResult(p *print.Printer, outputFormat string, instances []sqlserverflex.InstanceListInstance) error {
+	switch outputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(instances, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal SQLServer Flex instance list: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	case print.YAMLOutputFormat:
+		details, err := yaml.MarshalWithOptions(instances, yaml.IndentSequence(true))
+		if err != nil {
+			return fmt.Errorf("marshal SQLServer Flex instance list: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	default:
+		table := tables.NewTable()
+		table.SetHeader("ID", "NAME", "STATUS")
```

Comment:
```suggestion
		table.SetHeader("ID", "NAME", "STATE")
```

for uniformity with the other services

Reply:
For the other Flex databases we also have "status", as that's the field name in the JSON response. Same for Argus instance for example

Reply:
Alright, I checked and on the Portal it's also "Status"